Orange-OpenSource/Orange-Boosted-Bootstrap
# chore(merge main): patched commit → 58b1be9

- Firefox ESR
  - IE11 (v4 only)
  - Latest Edge, Chrome, Firefox, Safari
  - iOS Safari
  - Chrome & Firefox on Android
  - if bumping a minor or major version:
    - [x] Add docs version to `_data/docs-versions.yml`
    - [x] Manually change `docs_version` in `_config.yml`
    - [x] Update redirects in docs frontmatter
    - [x] Move docs to the new docs' subfolder?
  - check wrong matches in `CHANGELOG.md`, and maybe `site/content/docs/<version>/migration.md`
  - install [Conventionnal Changelog](https://github.com/conventional-changelog/conventional-changelog) and `conventional-changelog-cli` globally
  - run `conventional-changelog -p angular -i CHANGELOG.md -s`
  - and probably maintain [a ship list (eg for v4.4.0)](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/226)
- [ ] merge (on `v5-dev`)
  - attach zip file
  - paste CHANGELOG / Ship list in the release's description
  - if you are already logged in NPM (with a personnal account, for example), [you'd better use a repository scoped `.npmrc` file](https://stackoverflow.com/questions/30114166/how-to-have-multiple-npm-users-set-up-locally)
